enhancements/etcd/storage-migration-for-etcd-encryption.md

```diff
@@ -0,0 +1,182 @@
+---
+title: storage-migration-for-etcd-encryption
+authors:
+  - "@sanchezl"
+reviewers:
+  - "@sttts"
+  - "@deads2k"
+  - "@enj"
+approvers:
+  - TBD
+creation-date: 2019-09-11
+last-updated: 2019-09-11
+status: provisional
+---
+
+# storage-migration-for-etcd-encryption
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Introduce the `openshift-kube-storage-version-migrator-operator` to install the `StorageVersionMigration` CRD and the corresponding  `kube-storage-version-migrator` controller.  `StorageVersionMigration` resources can then be created by components, (initially  `kube-apiserver` and `openshift-apiserver`) to kick off storage migrations for resources.
```

Comment:
nit: StorageVersionMigration CRD provided by the kube-storage-version-migrator controller.

In other words, this is an upstream API.

Reply:
Put a link to an example StorageVersionMigration object or even paste it here.

Reply:
Done